server/statistics/hot_peer_cache.go

```diff
@@ -28,12 +28,22 @@ const (
 
 	rollingWindowsSize = 5
 
-	hotWriteRegionMinBytesRate = 1 * 1024
-	hotReadRegionMinBytesRate  = 8 * 1024
-
 	hotRegionReportMinInterval = 3
 
-	hotRegionAntiCount = 1
+	hotRegionAntiCount = 2
```

Comment:
Is it changed on purpose?

Reply:
Yes, see line 291, I change the `<` to `<=`.